homeassistant/components/squeezebox/coordinator.py

```diff
@@ -70,3 +74,46 @@ def _prepare_status_data(self, data: dict) -> dict:
 
         _LOGGER.debug("Processed serverstatus %s=%s", self.lms.name, data)
         return data
+
+
+class SqueezeBoxPlayerUpdateCoordinator(DataUpdateCoordinator):
+    """Coordinator for Squeezebox players."""
+
+    def __init__(self, hass: HomeAssistant, player: Player, server_uuid: str) -> None:
+        """Initialize the coordinator."""
+        super().__init__(
+            hass,
+            _LOGGER,
+            name=player.name,
+            update_interval=timedelta(seconds=PLAYER_UPDATE_INTERVAL),
+            always_update=True,
+        )
+        self.player = player
+        self.available = True
+        self._remove_dispatcher: Callable | None = None
+        self.server_uuid = server_uuid
+
+    async def _async_update_data(self) -> dict:
+        """Update the Player() object."""
```

Comment:
I find this docstring to be strange

Reply:
Yes, it could be more descriptive. We call the async_update() method of the Player object to force an update, but we do other things too. I'll change it.